app/features/notifications/strategies/CanShowNotificationsOnScreenStrategy.js

```diff
@@ -0,0 +1,12 @@
+import moment from "moment";
+import { Actions } from "react-native-router-flux";
+
+import { SCENES_TO_NOT_RENDER_NOTIFICATIONS } from '../constants';
+
+function CanShowNotificationsOnScreenStrategy() {
```

Comment:
Let's just create function canShowNotificationOnCurrentScreen in utils and use it directly in app-service.

Reply:
Got it. Will do